### PR TITLE
[GITHUB-20] Fix PECL, -dev Installations

### PR DESCRIPTION
**Pull request by dennisconrad**

### CODE REVIEW

File: .version [hunk 1]
```diff
@@ -1 +1 @@
-v2.1
+v2.2.0
```
Comment (lobsterdore) [inline .version]: Should just be v2.2, the release script in the Sansible repo controls the patch level.

File: tasks/pecl.yml [hunk 2]
```diff
@@ -9,7 +9,11 @@
...
-  changed_when: php_pecl_result.rc == 0
-  failed_when: not php_pecl_result.stdout is search("already installed") or
-               not php_pecl_result.stdout is search("install ok:")
+  changed_when: php_pecl_result.stdout is search("install ok") or
+                php_pecl_result.stderr is search("install ok")
+  failed_when:
+    - not php_pecl_result.stdout is search("already installed")
+    - not php_pecl_result.stdout is search("install ok")
+    - not php_pecl_result.stderr is search("already installed")
+    - not php_pecl_result.stderr is search("install ok")
```
Comment (lobsterdore) [inline tasks/pecl.yml]: Would the Ansible PEAR module handle this for us?

Comment (dennisconrad) [reply tasks/pecl.yml]: It does indeed once you prefix the package name with `pecl/` :-)

**Comment by lobsterdore** [comment]
LGTM 👍

**Pull request merged**